components/bl-double-calendar-component/src/specific-date-range.js

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+
+import { addDays } from './lib/date-fns.min.js';
+import DatePicker from './lib/react-datepicker.min.js';
+
+export function SpecificDateRange({ selectionRange, defaultSelectedDate, onDateSelect }) {
+  const [selectedDate, setSelectedDate] = useState(defaultSelectedDate ? new Date(defaultSelectedDate) : new Date());
```

Comment:
as a variant, leave what you like

`defaultSelectedDate ? new Date(defaultSelectedDate) : new Date()`
to
`new Date(defaultSelectedDate || Date.now())`